paper-input-behavior.html

```diff
@@ -335,6 +346,50 @@
     },
 
     /**
+     * Forward focus to inputElement
+     */
+    _onFocus: function() {
+      if (!this._shiftTabPressed) {
+        // iron-autogrow-textarea
+        if (this.inputElement.textarea) {
```

Comment:
Additionally, if you created an accessor (`this._focusableElement`, for example), you would only need to override the access in sub-implementations, rather than all the methods that deal with that element.